index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css"/>
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="header">
+        <div class=""></div>
+        <div class="">
+            <a href="" class="logo">Alexis UWIMANA</a>
```

Comment:
- [ ] Please ensure to follow the design as provided in the figma design. Your logo is on the left side of the page but it looks a bit off since it displaces the hamburger button. Make the logo a bit smaller (Probably should cover about 30% of the total width).